integrations/jenkins/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+
+# 0.2.0 (2024-01-22)
+
+### Features
+
+- Added support for ingesting users (PORT-6268).
+
+### Improvements
+
+- Increase page size for jobs and build to 50.
```

Comment:
I don't think this is necessary. Remove it

Reply:
Alright.